pkg/monitoring/metrics/virt-controller/vmstats_collector.go

```diff
@@ -20,27 +20,37 @@
 package virt_controller
 
 import (
+	"fmt"
+
 	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	k8sv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
-	"kubevirt.io/client-go/log"
-
 	k6tv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/log"
 )
 
 var (
 	vmStatsCollector = operatormetrics.Collector{
-		Metrics:         append(timestampMetrics, vmResourceRequests, vmInfo),
+		Metrics:         append(timestampMetrics, vmResourceRequests, vmInfo, vmPVCAllocatedSize),
 		CollectCallback: vmStatsCollectorCallback,
 	}
 
+	vmPVCAllocatedSize = operatormetrics.NewGaugeVec(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_vm_pvc_allocated_size_bytes",
+			Help: "Allocated persistent volume claim size in bytes for each Virtual Machine.",
+		},
+		[]string{"name", "namespace", "disk_name", "pvc_name"},
+	)
```

Comment:
maybe he should try to keep the order in `Metrics:         append(timestampMetrics, vmResourceRequests, vmInfo, vmPVCAllocatedSize),` going forward

please create this metric last

Reply:
so I update all metric to be according to the order